clang-tools-extra/clang-tidy/modernize/UseEqualsDeleteCheck.h

```diff
@@ -13,22 +13,9 @@
 
 namespace clang::tidy::modernize {
 
-/// Mark unimplemented private special member functions with '= delete'.
-/// \code
-///   struct A {
-///   private:
-///     A(const A&);
-///     A& operator=(const A&);
-///   };
-/// \endcode
-/// Is converted to:
-/// \code
-///   struct A {
-///   private:
-///     A(const A&) = delete;
-///     A& operator=(const A&) = delete;
-///   };
-/// \endcode
+/// Identifies unimplemented private special member functions, and recommends
+/// using ``= delete`` for them, as well as relocating them from the ``private``
+/// to the ``public`` section.
```

Comment:
looks like check currently does not limit private -> public to only special member functions, in such case we could write:
`as well as relocating deleted functions from the ``private`` to the ``public`` section.`